flink-yarn/src/test/java/org/apache/flink/yarn/YarnClusterDescriptorTest.java

```diff
@@ -912,4 +913,17 @@ private Map<String, String> getTestMasterEnv(
                     appId.toString());
         }
     }
+
+    @Test
+    public void testByteDesc() {
+        long bytesInMB = 1024 * 1024;
+        // 128 MB
+        assertThat(StringUtils.byteDesc(bytesInMB * 128)).isEqualTo("128 MB");
+        // 512 MB
+        assertThat(StringUtils.byteDesc(bytesInMB * 512)).isEqualTo("512 MB");
+        // 1 GB
+        assertThat(StringUtils.byteDesc(bytesInMB * 1024)).isEqualTo("1 GB");
+        // 128 GB
+        assertThat(StringUtils.byteDesc(bytesInMB * 131072)).isEqualTo("128 GB");
+    }
```

Comment:
Flink unit test shouldn't check the feature of others services, such as: yarn, hdfs, zookeeper and jdk.

Flink unit test should check the feature in the flink side.

Reply:
Thank you for your suggestion!